constraint_utils/constraint_basics.py

```diff
@@ -843,12 +925,43 @@ def oe_confidence_interval(ht: hl.Table, obs: hl.expr.Int32Expression, exp: hl.e
 
 
 def calculate_z(input_ht: hl.Table, obs: hl.expr.NumericExpression, exp: hl.expr.NumericExpression, output: str = 'z_raw') -> hl.Table:
+    """
+    Compute the signed raw z score using observed and expected variant counts.
+    
+    The raw z scores are positive when the transcript had fewer variants than expected, and are negative when transcripts had more variants than expected.
+    
+    Function will have following annotations in the output Table in addition to keys:
+        - `output` - the raw z score
+
+    :param input_ht: Input Table.
+    :param obs: Observed variant counts.
+    :param exp: Expected variant counts.
```

Comment:
These are actually expressions
```suggestion
    :param obs: Observed variant count expression.
    :param exp: Expected variant count expression.
```